docs/usage.md

```diff
@@ -168,26 +168,29 @@ bin/thor -h
 | `--api-call-gas-limit`      | Limit contract call gas (default: 50000000)                                                 |
 | `--api-backtrace-limit`     | Limit the distance between 'position' and best block for subscriptions APIs (default: 1000) |
 | `--api-allow-custom-tracer` | Allow custom JS tracer to be used for the tracer API                                        |
+| `--enable-api-logs`         | Enables API requests logging                                                                |
 | `--verbosity`               | Log verbosity (0-9) (default: 3)                                                            |
 | `--max-peers`               | Maximum number of P2P network peers (P2P network disabled if set to 0) (default: 25)        |
 | `--p2p-port`                | P2P network listening port (default: 11235)                                                 |
 | `--nat`                     | Port mapping mechanism (any\|none\|upnp\|pmp\|extip:<IP>) (default: "any")                  |
+| `--bootnode`                | Comma separated list of bootnode IDs                                                        |
+| `--import`                  | Import master key from keystore                                                             |
```

Comment:
import and export are for the master key command, not the node

Reply:
thanks going to remove it